### DISCUSSION TITLE
elastic agent - send to multiple destinations ?

### DISCUSSION THREAD

**Discussion by 73F0SG1R** [Security-Onion-Solutions/securityonion#11501] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

32

### Storage for /

200

### Storage for /nsm

500

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

can the elastic agent be configured to send to multiple destinations? (securityonion, cribl, data lake, etc.)?

thank you

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
Hello there!

Different logs to different locations or output all logs two different locations?

**Comment by 73F0SG1R** [comment]
i guess it could be either, but all logs to 2 location (say siem and datalake, etc.)

**Reply by defensivedepth** [reply, marked as answer]
This is not supported directly in Elastic Agent, but once you get the logs shipped to Security Onion, you can forward them elsewhere: https://docs.securityonion.net/en/2.4/logstash.html#forwarding-events-to-an-external-destination
